ajax/libs/bootstrap-iconpicker/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "bootstrap-iconpicker",
+  "version": "1.8.2",
+  "filename": "js/bootstrap-iconpicker.min.js",
+  "description": "A simple iconpicker for Bootstrap 3.x",
+  "homepage": "https://victor-valencia.github.io/bootstrap-iconpicker/",
+  "keywords": [
+    "bootstrap",
+    "iconpicker",
+    "icon-fonts",
+    "icons",
+    "css",
+    "javascript",
+    "jquery"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/victor-valencia/bootstrap-iconpicker.git",
+    "fileMap": [
+      {
+        "basePath": "bootstrap-iconpicker",
+        "files": [
+          "**/*"
```

Comment:
@victor-valencia I found that you release new version v1.9.0. But the file structure is different from old versions.
I think there are two ways can do:
1. After this PR merge, we will update the auto-update config as the following and add v1.9.0.
```
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/victor-valencia/bootstrap-iconpicker.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
    ]
  },
```

2. Just add the v1.9.0 in this PR. After this PR merge, we'll Manually add old versions before v1.8.2.

Which one you'd like to?

Thank you.